### PR TITLE
outcomes list and mobile order form

### LINKED ISSUE #3769: condense outcome list

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/3769

closes https://github.com/AugurProject/augur/issues/3769

### CODE REVIEW

File: packages/augur-ui/src/modules/market/components/market-outcomes-list/market-outcomes-list.styles.less [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @import (reference) '~assets/styles/shared';
 
-.OutcomesList {}
+.OutcomesList {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/market-outcomes-list/market-outcomes-list.styles.less]: Is this needed?

Comment (phoebemirman) [reply packages/augur-ui/src/modules/market/components/market-outcomes-list/market-outcomes-list.styles.less]: no

**Review by bthaile — APPROVED** [review]
lgtm

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🤙

 This link will always point to this PR's latest changes: https://peek.run/o9phx5e

**Pull request merged**